src/librustc_codegen_llvm/context.rs

```diff
@@ -227,6 +227,16 @@ pub unsafe fn create_module(
         llvm::LLVMRustAddModuleFlag(llmod, avoid_plt, 1);
     }
 
+    // Set module flags to enable Windows Control Flow Guard (/guard:cf) metadata
+    // only (cfguard=1) or metadata and checks (cfguard=2).
+    match sess.opts.debugging_opts.control_flow_guard {
+        CFGuard::NoChecks => {
+            llvm::LLVMRustAddModuleFlag(llmod, "cfguard\0".as_ptr() as *const _, 1)
+        }
+        CFGuard::Checks => llvm::LLVMRustAddModuleFlag(llmod, "cfguard\0".as_ptr() as *const _, 2),
+        _ => {}
```

Comment:
Make this match exhaustive (i.e. `s/_/CFGuard::Disabled/`)

Reply:
Fixed